docs/sources/flow/reference/config-blocks/import.git.md

```diff
@@ -48,8 +48,10 @@ with a `git clone REPOSITORY_ADDRESS` command, such as `https://github.com/grafa
 When provided, the `revision` attribute must be set to a valid branch, tag, or
 commit SHA within the repository.
 
-You must set the `path` attribute to a path accessible from the repository's root,
-such as `FILE_NAME.river` or `FOLDER_NAME/FILE_NAME.river`.
+You must set the `path` attribute to a path accessible from the repository's root.
+It can either be a river file such as `FILE_NAME.river` or `DIR_NAME/FILE_NAME.river` or
+a directory containing river files such as `DIR_NAME` or `.` if the river files are stored at the root
+of the repository.
```

Comment:
River is a proper noun:

```suggestion
You must set the `path` attribute to a path accessible from the repository's root.
It can either be a River file such as `FILE_NAME.river` or `DIR_NAME/FILE_NAME.river` or
a directory containing River files such as `DIR_NAME` or `.` if the River files are stored at the root
of the repository.
```

Reply:
I missed that in the review :-(